engine.cfg

```diff
@@ -1,13 +1,19 @@
 [application]
 
-icon="icon.png"
-main_scene="res://main.scn"
 name="sanic-gaiden"
+main_scene="res://main.scn"
+icon="icon.png"
+
+[autoload]
+
+globals="*res://scripts/globals.gd"
```

Comment:
Looks like the `*` at the beginning of the path is causing the path to be invalid and crash for me. Removing it seems to fix it.